azure_monitor/src/azure_monitor/protocol.py

```diff
@@ -146,30 +209,39 @@ def to_dict(self):
 
 
 class ExceptionData(BaseObject):
+    """An instance of Exception represents a handled or unhandled exception that
+    occurred during execution of the monitored application.
+
+    Args:
+        ver: Schema version.
+        exceptions: Exception chain - list of inner exceptions.
+        severity_level: Severity level. Mostly used to indicate exception severity
+        level when it is reported by logging library.
+        properties: Collection of custom properties.
+        measurements: Collection of custom measurements.
+    """
+
     __slots__ = (
         "ver",
         "exceptions",
         "severity_level",
-        "problem_id",
         "properties",
         "measurements",
     )
 
     def __init__(
         self,
-        ver=2,
-        exceptions=None,
-        severity_level=None,
-        problem_id=None,
-        properties=None,
-        measurements=None,
+        ver: int = 2,
+        exceptions: any = None,
+        severity_level: int = None,
+        properties: typing.Dict[str, any] = None,
+        measurements: any = None,
```

Comment:
I believe `measurements` is a dict.

https://github.com/microsoft/ApplicationInsights-Home/blob/master/EndpointSpecs/Schemas/Bond/ExceptionData.bond#L31